report-flat/src/App.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div id="app">
+    <ReportOptions :options="options"/>
+    <div class="suite">
+      <hr/>
+      <img id="logo" src="./assets/logo.png">
+      Company: {{suite.company}} Project: {{suite.project}} Name: {{suite.name}} CorrelationID:
+      {{suite.correlationId}}
+      <hr/>
+      <div class="test" v-for="test in suite.tests">
+        <span class="testName"/> TEST: {{test.name}}
+        <div class="url" v-for="url in test.urls">
+          <span class="domain">{{url.domain}}</span><span class="urlname">{{url.name}}</span>
+          <br>
+          <div class="step" v-for="step in url.steps">
+            <AetStep :options="options" :model="step"></AetStep>
+          </div>
+        </div>
+        <hr/>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  import AetStep from './components/AetStep';
+  import ReportOptions from './components/ReportOptions';
+
+  export default {
+    name: 'app',
+    components: {
+      AetStep, ReportOptions
+    },
+
+    data() {
+      return {
+        suite: Object,
+        options: {
+          showPattern: true,
+          showCollected: true,
+          showMask: true,
+          showJS: true,
+          showScreen: true
+        },
+      };
+    },
+    created: function () {
```

Comment:
Inconsistency in formating, new lines between members of object/or no new lines, pick one. The same applies to the way you define new functions